packages/fast-components-react-base/src/radio/radio.tsx

```diff
@@ -0,0 +1,91 @@
+import * as React from "react";
+import * as ReactDOM from "react-dom";
+import Foundation, { HandledProps } from "@microsoft/fast-components-foundation-react";
+import { RadioHandledProps, RadioManagedClasses, RadioProps, RadioUnhandledProps } from "./radio.props";
+import { ManagedClasses, RadioClassNameContract } from "@microsoft/fast-components-class-name-contracts-base";
+import { get } from "lodash-es";
+
+/**
+ * Radio slot options
+ */
+export enum RadioSlot {
+    label = "label"
+}
+
+class Radio extends Foundation<RadioHandledProps & RadioManagedClasses, RadioUnhandledProps, {}> {
```

Comment:
You can remove `RadioManagedClasses` after you make changes to the prop interfaces.